spec/controller_spec_helper.rb

```diff
@@ -257,3 +257,80 @@ def split_date_to_params(key, date)
     :"#{key}_meridian" => date.strftime("%p"),
   }
 end
+
+def it_should_support_purchasing
```

Comment:
I would lean towards using a shared example for this rather than a big method. And then I would put that in `spec/support/shared_examples`. I generally dislike most of the methods defined in this helper. They're mainly an older style of testing that we try to avoid doing anymore.

Reply:
Also, we're tending towards moving to more active wording in our specs to go along with `should` -> `expect`, so this would be something like "it supports purchasing". Looks like you're doing it in the actual spec descriptions, so we should probably do it for these descriptions as well.

Reply:
I didn’t know what the Rspec-sanctioned way to do these is, so I’ll change them! (With Minitest, I would have just created a module, but that doesn’t quite work here)